WixInstaller/XenCenter.wxs

```diff
@@ -238,7 +238,7 @@
             <Publish Dialog="ExitDialog" 
                 Control="Finish" 
                 Event="DoAction" 
-                Value="LaunchApplication">WIXUI_EXITDIALOGOPTIONALCHECKBOX = 1 AND NOT (WixUI_InstallMode="Remove")
+                Value="LaunchApplication">WIXUI_EXITDIALOGOPTIONALCHECKBOX = 1 AND NOT (WixUI_InstallMode="Remove") AND XS_WixUIRMPressedOk="0"
```

Comment:
FYI. Technically, if `(WixUI_InstallMode="Remove") AND XS_WixUIRMPressedOk="0"` is `true`, the checkbox won't show at all. This is just an extra check